my-app/src/containers/channelPage.js

```diff
@@ -1,14 +1,22 @@
 import React from 'react';
 import Header from '../components/Header'
 import Body from '../components/Body'
+import { Link } from 'react-router-dom'
 
 const ChannelPage = (props) => {
   console.log(props)
     return (
```

Comment:
`props.location.data.channel.channelName`

it is quite a long thing to be typing out you can destructure the values.

```
const { channelName, channelIcon, programme: {contentUrl, title, desc} } = props.location.data.channel
```

now I can just use the values by name:

```html
<h1>{title}</h1>
<h2>{desc}</h2>
<img src={contentUrl} />
```

Its easier to read